### PR TITLE
chore: adjust layout of invite screen

### PR DESCRIPTION
**Pull request by ice-atlas**
- improve positioning of central image

### CODE REVIEW

File: src/screens/InviteFlow/InviteShare/components/InviteShareCard/components/ShareCard/index.tsx [hunk 1]
```diff
@@ -48,24 +42,22 @@ export const ShareCard = <P,>({buttons, onButtonPress, children}: Props<P>) => {
 };
 
 const styles = StyleSheet.create({
-  shareCard: {
-    position: 'absolute',
-    left: 0,
-    bottom: 0,
-    right: 0,
+  container: {
+    borderTopLeftRadius: rem(30),
```
Comment (ice-hades) [inline src/screens/InviteFlow/InviteShare/components/InviteShareCard/components/ShareCard/index.tsx]: move it to a const CONTAINER_TOP_RADIUS

**Review by ice-orion — DISMISSED** [review]

File: src/screens/InviteFlow/InviteShare/index.tsx [hunk 2]
```diff
@@ -49,15 +46,15 @@ const styles = StyleSheet.create({
...
   },
   icon: {
-    marginLeft: isRTL ? 0 : ICON_LEFT_OFFSET,
-    width: ICON_WIDTH,
-    height: ICON_WIDTH * 0.858,
-    marginTop: rem(17),
+    marginVertical: rem(16),
+    marginStart: rem(21),
+    flex: 1,
+    width: undefined,
```
Comment (ice-orion) [inline src/screens/InviteFlow/InviteShare/index.tsx:62]: Just curious why we need this?

File: src/screens/InviteFlow/InviteShare/index.tsx [hunk 3]
```diff
@@ -49,15 +46,15 @@ const styles = StyleSheet.create({
     borderTopRightRadius: rem(30),
   },
   description: {
-    ...font(15, null, 'regular', 'white', 'center'),
     marginTop: rem(45),
     marginHorizontal: rem(28),
+    ...font(15, null, 'regular', 'white', 'center'),
   },
   icon: {
-    marginLeft: isRTL ? 0 : ICON_LEFT_OFFSET,
```
Comment (ice-hector) [inline src/screens/InviteFlow/InviteShare/index.tsx:57]: Why we remove isRTL check. Did you check the rtl/not rtl layout on both platforms, does it work properly without the check and previous adjustments?

Comment (ice-atlas) [reply src/screens/InviteFlow/InviteShare/index.tsx:57]: used `marginStart`;)

File: src/screens/InviteFlow/InviteShare/components/InviteShareCard/components/ShareCard/index.tsx (hunk 1 above)
Comment (ice-atlas) [reply src/screens/InviteFlow/InviteShare/components/InviteShareCard/components/ShareCard/index.tsx]: done

File: src/screens/InviteFlow/InviteShare/index.tsx (hunk 2 above)
Comment (ice-atlas) [reply src/screens/InviteFlow/InviteShare/index.tsx:62]: added comment. Otherwise image does not occupy all width of the screen

**Review by ice-orion — APPROVED** [review]

**Review by ice-hades — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Pull request merged**